tests/Feature/Store/SearchPageTest.php

```diff
@@ -263,10 +263,11 @@ public function itPaginatesWhenRequired()
         ]);
 
         // Visit search page, make sure next page link is present and works
+        // <a class="page-link" href="http://arcv-store.test/registrations?page=2" rel="next" aria-label="Next »">›</a>
         $this->actingAs($centreUser, 'store')
             ->visit(URL::route('store.registration.index'))
-            ->see('<a href="' . URL::route('store.base') . '/registrations?page=2' . '" rel="next">»</a>')
-            ->click('»')
+            ->see('<a class="page-link" href="' . URL::route('store.base') . '/registrations?page=2' . '" rel="next" aria-label="Next »">›</a>')
```

Comment:
Is this a change that was part of the upgrade?

Reply:
Yes, well it broke the tests and I didn't change anything else.  It was the first thing I hit

Reply:
It's the paginator here:
![image](https://user-images.githubusercontent.com/861172/83853385-1fe90600-a70d-11ea-9a55-35357d2934e9.png)

Probably a library that updated the default design to use `<` instead of `<<`.  Probably no one will notice but worth mentioning at least to @Nikomus to see if he thinks it significantly affects the UX.

I think we can override those in a lang file.

Reply:
It's probably clearer as one chevron as it's the next page link rather than a last page link.